core/src/main/resources/defaults/blueprints/7.2.10/cdp-opdb.bp

```diff
@@ -313,7 +324,8 @@
           "yarn-JOBHISTORY-BASE",
           "yarn-RESOURCEMANAGER-BASE",
           "yarn-GATEWAY-BASE",
-          "zookeeper-SERVER-BASE"
+          "zookeeper-SERVER-BASE",
+          "omid-OMID_TSO-BASE"
```

Comment:
Can you double-check the max heap which is allocated for each process on the leader node? Want to make sure that we don't over-commit memory on this host.

CM tries to do this but is more-or-less guessing about it, IIRC.

Reply:
Memory usage seems to be fine, we still have more memory on the leader node than on the workers for example.
https://richard-opdb-omid-gateway0.richarda.xcu2-8y8x.dev.cldr.work/cmf/hardware/hosts/1546333375/status

Reply:
The leader node is an m5.2xlarge one, it has 32 GB memory.
According to CM the sum of max heap sizes (+ 30% Overhead for every service) is 18,8 Gb, but it writes 10,4 for the ZooKeeper server.
These values are collected from the configs set by blueprint or CM params.

Reply:
Perfect! Thanks for the great analysis.